src/Main.js

```diff
@@ -1,16 +1,26 @@
 import React from 'react';
+import data from './data.json'
 import Hornedbeast from './Hornedbeast';
-import bull from './Assests/bull.jpeg'
-import pronghorn from './Assests/pronghorn.jpeg'
-import elk from './Assests/elksnow.jpeg'
+
 
 class Main extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      beasts: data
+    }
+  }
   render () {
+    let animalArray = this.state.beasts.map((animal, index) => (
+      <Hornedbeast
+        src={animal.image_url}
```

Comment:
also here. Make it
```js
image_url={animal.image_url}
```
if each bit of data continues to have the same name as it's passed to each component, it gets way easier to keep track of your props.